src/core/datatypes/forte_wstring.h

```diff
@@ -52,13 +52,22 @@ class CIEC_WSTRING final : public CIEC_ANY_STRING {
       TForteWChar value = static_cast<TForteWChar>(paValue);
       auto &converter = std::use_facet<std::codecvt<char16_t, char, std::mbstate_t>>(std::locale());
       std::mbstate_t mb{};
+#ifdef _MSC_VER
```

Comment:
I don't think in this case it makes sense to have two versions. I would prefer to only have the one that compiles for all systems.